super-csv/src/main/java/org/supercsv/prefs/CsvPreference.java

```diff
@@ -430,21 +448,38 @@ public Builder maxLinesPerRow(final int maxLinesPerRow) {
 		}
 		
 		/**
-		 * Uses an EmptyColumnParsing to determine whether empty String (i.e. "") should be read as empty string instead as null. 
+		 * Uses an EmptyColumnParsing to determine whether empty String (i.e. "") should be read as empty string instead as null.
 		 * The default is <tt>ParseEmptyColumnsAsNull</tt>.
-		 * 
+		 *
 		 * @since 2.4.1
 		 * @param emptyColumnParsing
 		 *            the emptyColumnParsing
 		 * @return the updated Builder
-		 */		
+		 */
 		public Builder setEmptyColumnParsing(final EmptyColumnParsing emptyColumnParsing) {
 			if( emptyColumnParsing == null ) {
 				throw new NullPointerException("emptyColumnParsing should not be null");
 			}
 			this.emptyColumnParsing = emptyColumnParsing;
 			return this;
 		}
+
+		/**
+		 * Value indicating the character to use for escaping a quote char.
+		 *
+		 * NOTE: this field is only supported for reading data.  In writing data this value is
+		 * ignored and data is written in RFC-compliant form, meaning the quote character is also
+		 * the quote escape character.
+		 *
+		 * @since X
```

Comment:
@ZioberMichal @ash211  good question. If we are to opt-in on semantic versioning, the version will depend on breaking changes. ie. a breaking change will make the next release v3.0 rather than 2.5 - but I guess we are not yet using semantic versioning - although perhaps we should start doing it :)